frontend/packages/knative-plugin/integration-tests/features/knative/create-knative-service-for-Deployment-and-DeploymentConfig.feature

```diff
@@ -0,0 +1,77 @@
+
+        @regression
+        Scenario: Create serverless workload (ksvc) for existing Deployment from kebab menu of Deployments page
+            Given user has created deployment "nodejs-ex-git1"
+              And user is at Deployments page
+             When user clicks on kebab button "nodejs-ex-git1" to open kebab menu
+              And user selects Make Serverless option
+              And user clicks on Create button in Make Serverless form
+             Then user will be redirected to topology with knative workload "ksvc-nodejs-ex-git1"
+
+
+        @regression
+        Scenario: Create serverless workload (ksvc) for existing Deployment from context menu whose Route is not defined
+            Given user has created deployment "hello-openshift1" with no Route defined
+              And user is at Topology page
+             When user right clicks on "hello-openshift1" to open context menu
+              And user selects Make Serverless option
+              And user clicks on Create button in Make Serverless form
+             Then user will be redirected to topology with knative workload "ksvc-hello-openshift1"
+              And user can see Routes available in the Rsource tab of sidebar for knative workload "ksvc-hello-openshift1"
+
+
+        @smoke
+        Scenario Outline: Edit knative workload created from deployment
+            Given user has created deployment "nodejs-ex-git1" from Import from Git way
+              And user has created knative workload "ksvc-nodejs-ex-git1" from deployment "nodejs-ex-git1"
+              And user is at Topology page
+             When user right clicks on "ksvc-nodejs-ex-git1" to open context menu
+              And user selects Edit ksvc-nodejs-ex-git1 option
+              And user clicks on Scaling in Advanced option of Import from Git form
+              And user scales value of Concurrency utilization to 4
+              And user clicks on Save button
+              And user right clicks on the knative service workoad "ksvc-nodejs-ex-git1" in Topology page
+              And user selects option Edit service from the context menu
+             Then user is able to see value of "autoscaling.knative.dev/targetUtilizationPercentage" as 4
+
+
+        @regression
+        Scenario: Create serverless workload (ksvc) for existing Deployment having HPA associated with it from context menu
+            Given user has created deployment "nodejs-ex-git2"
+              And user has added HPA with CPU and Memory Utilization percentage as 40 and 60 percent respectively
+              And user is at Topology page
+             When user right clicks on "nodejs-ex-git1" to open context menu
+              And user selects Make Serverless option
+              And user clicks on Create button in Make Serverless form
+              And user user right clicks on newly created knative workload "ksvc-nodejs-ex-git2"
+              And user selects "Edit HorizontalPodAutoscaler" option from context menu
```

Comment:
I don't think this would be happening, essentially we are updating min/max replicas of KSVC based on any associated HPA
```suggestion
              And user selects "Edit HorizontalPodAutoscaler" option from context menu
```

Reply:
Updated